src/views/DatasetManager/components/DatasetOverview/components/DatasetActions/DatasetActions.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ButtonGroup from '@mui/material/ButtonGroup';
+import { DeleteDatasetButton } from '../../../DeleteDatasetButton/DeleteDatasetButton';
+
+export default function DatasetActions({ dataset }) {
+  return (
+    <ButtonGroup>
+      <DeleteDatasetButton dataset={dataset} location="DatasetActions" />
```

Comment:
Could we replace `location="DatasetActions"` by `location="dataset-actions-"`? This would make the resulting data-cy more consistent with other data-cy values (`dataset-actions-dataset-delete-button-<dataset_id>`)